### ISSUE TITLE
custome training error 

### ISSUE THREAD

**Issue by NarasimmanSaravana1994** [mozilla/DeepSpeech#2496]
@lissyx 

I Initializing variables...
I STARTING Optimization
Epoch 0 |   Training | Elapsed Time: 0:00:00 | Steps: 0 | Loss: 0.000000                                                                                          Fatal Python error: Segmentation fault

**Comment by kdavis-mozilla** [comment]
@NarasimmanSaravana1994 Could you please fill out the issue template? Otherwise we will be unable to help you.

**Comment by lissyx** [comment]
Given the few info, it's unlikely to be a bug on our side...

**Comment by NarasimmanSaravana1994** [comment]
@lissyx 

Command Executed:
CUDA_VISIBLE_DEVICES=0 python3 DeepSpeech_testing.py --lm_binary_path ./data/deepspeech_dataset/lm.binary --lm_trie_path ./data/deepspeech_dataset/trie --learning_rate 0.0001 --alphabet_config_path ./data/alphabet.txt --beam_width 1024 --use_seq_length false --remove_export true --notest --train_batch_size 30 --train_files ./data/deepspeech_dataset/18-10-19_final_purned.csv --dev_files ./data/deepspeech_dataset/11-10-19_en_test.csv --export_dir ./export  --epochs 10 --dropout_rate4 0.3 --dropout_rate5 0.3


Now that error is solved. But my training is run in CPU only 
1.I install deepspeech-gpu and tensorflow-gpu but it running in cpu 
2.No process is running (showing in nvidia-smi)

What i'm missing i don't know.
Please guild me where i'm missing or need to check.

**Comment by lissyx** [comment]
@NarasimmanSaravana1994 This is not a bug, please use Discourse for support. You are playing with CUDA visibility, please make sure you don't mask any GPU. Please also make sure that TensorFlow is able to see your CUDA setup: `python3 -c "import tensorflow as tf; tf.Session();"`.
